script/cibuild

```diff
@@ -22,16 +22,18 @@ function loadEnvironmentVariables(filePath) {
 }
 
 function readEnvironmentVariables() {
-  if (process.platform === 'win32') {
-    loadEnvironmentVariables(path.resolve('/jenkins/config/atomcredentials'));
-  } else if (process.platform === 'darwin') {
-    loadEnvironmentVariables('/var/lib/jenkins/config/atomcredentials');
-    loadEnvironmentVariables('/var/lib/jenkins/config/xcodekeychain');
-  } else if (process.platform === 'linux') {
-    // Use Clang for building native code, the GCC on Precise is too old.
-    process.env.CC = 'clang';
-    process.env.CXX = 'clang++';
-    process.env.npm_config_clang = '1';
+  if (process.env.JANKY_SHA1) {
```

Comment:
:sparkles: Was thinking of creating a PR for this to get rid of the ENOENT errors on Travis.